notebooks/olmocr-pdf-vlm/olmocr-pdf-vlm.ipynb

```diff
@@ -0,0 +1,594 @@
+{
```

Comment:
<p>do we really need this section?</p><p>If it is needed only for getting input image for next stage, I think it is better just save image somewhere and save it and add reference to original model card with how to preprocess pdf docs to images</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2842/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Really not, removed</p>